app/src/main/kotlin/com/wire/android/ui/main/conversation/CallScreen.kt

```diff
@@ -20,10 +20,10 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.wire.android.R
 import com.wire.android.ui.common.UserProfileAvatar
-import com.wire.android.ui.main.conversation.all.model.toUserInfoLabel
-import com.wire.android.ui.main.conversation.call.model.Call
-import com.wire.android.ui.main.conversation.call.model.CallEvent
-import com.wire.android.ui.main.conversation.call.model.CallTime
+import com.wire.android.ui.main.conversation.model.toUserInfoLabel
+import com.wire.android.ui.main.conversation.model.Call
+import com.wire.android.ui.main.conversation.model.CallEvent
+import com.wire.android.ui.main.conversation.model.CallTime
```

Comment:
Shouldn't all of these elements live under `conversations` instead of `conversation`? I'm saying that in order to avoid confusion for all the components regarding the incoming Conversation detail screen

Reply:
I'll leave it as it is for now and let's think about restructering it in my next PR with conversation detail screen